api-doc/openapi.yaml

```diff
@@ -1,48 +1,514 @@
+          $ref: '#/components/responses/IncentiveCodesOk'
+    post:
+      security:
+        - ApiKeyAuth: []
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/IncentiveCode'
+      responses:
+        '200':
+          $ref: '#/components/responses/OkEmpty'
+    delete:
+      security:
+        - ApiKeyAuth: []
+      responses:
+        '200':
+          $ref: '#/components/responses/OkEmpty'
 components:
   responses:
-    TargetOk:
+    OkEmpty:
       description: Donation target
+    DonationsAllOk:
+      description: List of donations
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/Donations'
+    DonationsOk:
+      description: Donation by id
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/Donation'
+    GameDataAllOk:
+      description: List of game data
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/GameDataAll'
+    GameDataOk:
+      description: Game data by id
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/GameData'
+    StreamMetadataOk:
+      description: Stream metadata
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/StreamMetadata'
+    PlayerInfoOk:
+      description: Player info
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/PlayerInfo'
+    IncentivesAllOk:
+      description: List of incentives
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/Incentives'
+    IncentivesOk:
+      description: Incentives by id (uuid)
       content:
         application/json:
           schema:
-            $ref: '#/components/schemas/Target'
+            $ref: '#/components/schemas/Incentive'
+    IncentiveCodesAllOk:
+      description: List of incentive code data
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/IncentiveCodes'
+    IncentiveCodesOk:
+      description: Incentice data by code
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/IncentiveCode'
   schemas:
-    Target:
-      type: object
-      required:
-        - target
-        - currency
-        - _links
-      properties:
-        _links:
-          type: object
-          required:
-            - href
-          properties:
-            href:
-              type: string
-              format: uri
-              description: A link to self
-        target:
+    Donation:
+      type: object
+      properties:
+        timestamp:
+          type: string
+        name:
+          type: string
+        message:
+          type: string
+        amount:
+          type: integer
```

Comment:
Float maybe? Or whatever it is in OpenAPI

Reply:
number, updated